src/vivarium_gates_mncnh/data/loader.py

```diff
@@ -314,6 +319,77 @@ def load_maternal_disorder_yld_rate(
     return yld_rate
 
 
+def load_lbwsg_rr(
```

Comment:
Is this stuff all copied from another repo? If so, can you create a ticket to move this code inside vivarium-inputs.

Reply:
Yes the docs linked the LBWSG RR interpolation from the NO project so I will make a ticket.